app/_locales/en/messages.json

```diff
@@ -1352,6 +1355,9 @@
     "message": "Stack:",
     "description": "Title for error stack, which is displayed for debugging purposes"
   },
+  "errorWhileConnectingToRPC": {
+    "message": "Error while connecting to the custom RPC."
+  },
```

Comment:
Can we use the terminology `Network` here and in the translation above to align with the terminology used for the other UI elements?